.changes/next-release/enhancement-eks gettoken-65000.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should get rid of this change log entry as it has already been accounted for in the Friday (5/6) release